package.json

```diff
@@ -7,7 +7,7 @@
     "start": "webpack-dev-server --open --config webpack.dev.js",
     "build": "webpack --config webpack.prod.js",
     "storybook": "start-storybook -p 6006 --ci -c .storybook",
-    "lint": "./node_modules/tslint/bin/tslint ./src/**/*",
+    "lint": "tslint --project tsconfig.json",
```

Comment:
What was happening here? CI?

Reply:
This was the Windows-specific bug I was talking about yesterday...  There was a linting error in the CI, but I couldn't test it locally because Windows fails on the ./ notation.